src/contexts/providers.context.tsx

```diff
@@ -86,7 +86,8 @@ const ProvidersProvider: FC<PropsWithChildren> = (props) => {
               if (!supportedChainIds.includes(requestedChainId)) {
                 return setAccount({
                   status: "failed",
-                  error: "Switch your network to Ethereum or Polygon zkEVM to continue",
+                  error:
+                    "Switch your network to Goerli Ethereum testnet or Polygon zkEVM to continue",
```

Comment:
Instead of hardcoding the `testnet` literal, we should infer it from the chainId.

We could use this enum declaring the Ethereum chainIds (we should remove `RINKEBY` by the way):

```typescript
export enum EthereumChainId {
  MAINNET = 1,
  RINKEBY = 4,
  GOERLI = 5,
}
```

Or maybe we can use the helpers `getNetworkName` and `getDeploymentName`.